tableau-databricks/connection-fields.xml

```diff
@@ -23,6 +23,10 @@ limitations under the License. -->
     </field>
     <!-- /Endpoint -->
 
+    <field name="v-http-path" label="@string/http_path_prompt/" category="endpoint" value-type="string">	
+        <!-- validation-rule reg-exp="^[0-9A-Za-z-_/]*$" /-->
```

Comment:
we can keep this line, but the indentation is wrong

Reply:
Have we decided that want to enable verification here or not?

Reply:
Similarly concerning the Server Hostname validation rule: if a company has configured "mycompanyname-databricks.com" to redirect to "adb-94384578309420.databricks.com", then using the first URL would not work in Tableau. Do you think the validation is needlessly restrictive?

Reply:
Or actually, never mind about the Server Hostname validation. I misread the regex.

Reply:
I don't remember why I disabled http path validation here. Does it break if it's enabled?

Reply:
Oh wait, actually it was disabled before my change.

Reply:
HTTP Path wasn't validated previously as it was simply the `<database>` field.